src/olympia/activity/models.py

```diff
@@ -212,6 +213,16 @@ class IPLog(ModelBase):
 
     activity_log = models.ForeignKey('ActivityLog', on_delete=models.CASCADE)
     ip_address = models.CharField(max_length=45)
+    ip_address_binary = VarBinaryField(max_length=16, null=True)
```

Comment:
I guess there isn't a django field that stores the ip address in an SQL compatible way and also has a nice string representation/setter in django?  (It'd be nice to have a single ip address field)

Reply:
If there isn't, maybe we could implement one on top of `VarBinaryField` as a follow-up?

Reply:
There isn't. That's not a bad idea...